prober/notification.go

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package prober
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"expvar"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+var (
+	alertGenerated   = expvar.NewInt("alert_generated")
+	alertFailed      = expvar.NewInt("alert_failed")
+	warningGenerated = expvar.NewInt("warning_generated")
+	warningFailed    = expvar.NewInt("warning_failed")
+)
+
+// sendAlert sends an alert to the incident response system, to
+// page a human responder immediately.
+// summary should be short and state the nature of the emergency.
+// details can be longer, up to 29 KBytes.
+func sendAlert(summary, details string) error {
+	type squadcastAlert struct {
+		Message     string            `json:"message"`
+		Description string            `json:"description"`
+		Tags        map[string]string `json:"tags,omitempty"`
+		Status      string            `json:"status"`
+		EventId     string            `json:"event_id"`
+	}
+
+	tags := map[string]string{"severity": "critical"}
+	sqa := squadcastAlert{
+		Message:     summary,
+		Description: details,
+		Tags:        tags,
```

Comment:
Suggestion:
```suggestion
	sqa := squadcastAlert{
		Message:     summary,
		Description: details,
		Tags:        map[string]string{"severity": "critical"},
```
(since we never use it again otherwise)